veros/settings.py

```diff
@@ -36,6 +36,7 @@
     ("enable_conserve_energy", Setting(True, bool, "exchange energy consistently")),
     ("enable_store_bottom_friction_tke", Setting(False, bool, "transfer dissipated energy by bottom/rayleig fric. to TKE, else transfer to internal waves")),
     ("enable_store_cabbeling_heat", Setting(False, bool, "transfer non-linear mixing terms to potential enthalpy, else transfer to TKE and EKE")),
+    ("enable_noslip_latteral", Setting(False, bool, "enable no-slip boundary conditions")),
```

Comment:
If you can, you are more than welcome to give a more detailed explanation. The minimal documentation of the other settings is inherited from PyOM, and nothing that I am particularly proud of.